pkg/metrics/smart/metrics.go

```diff
@@ -31,17 +31,56 @@ type MetricsData struct {
 	// blockDeviceTemperature is the temperature of the the blockdevice if it is reported
 	blockDeviceCurrentTemperature *prometheus.GaugeVec
 
+	// blockDevicehighestTemperature is the highest temperature of the the blockdevice if it is reported
+	blockDeviceHighestTemperature *prometheus.GaugeVec
+
+	// blockDevicelowestTemperature is the highest temperature of the the blockdevice if it is reported
+	blockDeviceLowestTemperature *prometheus.GaugeVec
+
+	// blockDeviceRotationalLatency of block devices
+	blockDeviceRotationalLatency *prometheus.GaugeVec
+
+	// blockDeviceCapacity is capacity of block devices
+	blockDeviceCapacity *prometheus.GaugeVec
+
+	//blockDeviceLogicalSectorSize is logical sector size of block devices in bytes
+	blockDeviceLogicalSectorSize *prometheus.GaugeVec
+
+	//blockDevicePhysicalSectorSize is logical sector size of block devices in bytes
+	blockDevicePhysicalSectorSize *prometheus.GaugeVec
+
+	//blockDeviceRotationRate is rotation rate of a block device in RPM
+	blockDeviceRotationRate *prometheus.GaugeVec
+
+	//blockDeviceTotalReadBytes is the total number of bytes read from the block device
+	blockDeviceTotalReadBytes *prometheus.GaugeVec
+
+	//blockDeviceTotalWrittenBytes is the total number of bytes written from the block device
+	blockDeviceTotalWrittenBytes *prometheus.GaugeVec
+
+	//blockDeviceUtilizationRate is utilization rate of the block device
+	blockDeviceUtilizationRate *prometheus.GaugeVec
+
+	// blockDevicePercentEnduranceUsed
+	blockDevicePercentEnduranceUsed *prometheus.GaugeVec
+
 	// errors and rejected requests
 	rejectRequestCount prometheus.Counter
 	errorRequestCount  prometheus.Counter
 }
 
 //MetricsLabels are the labels that are available on the prometheus metrics
 type MetricsLabels struct {
-	UUID     string
-	Path     string
-	HostName string
-	NodeName string
+	UUID            string
+	Path            string
+	HostName        string
+	NodeName        string
+	ModelNumber     string
+	Model           string
+	SerialNumber    string
+	VendorID        string
+	FirmwareVersion string
+	DriveType       string
```

Comment:
Why are the labels changed? The older labels were selected so as to connect/merge metrics from node exporter and NDM exporter. 
i.e Those labels help to map a `blockdevice-xxx` resource to corresponding metrics from prometheus.